index.js

```diff
@@ -184,7 +61,7 @@ class BookCollection {
 }
 
 const arrayCollection = new BookCollection(
-  JSON.parse(localStorage.getItem('bookData'))
+  JSON.parse(localStorage.getItem('bookData')),
 );
```

Comment:
- [ ] Kindly you still have a problem, because you are depending on getting the data from local storage, but on the reviewer device this data does not exist, that's why we are having the issue and you don't.

The error
![image](https://user-images.githubusercontent.com/35535130/159445192-28a1c72a-41ea-458a-b4cd-4d0fd20cb628.png)


To solve this issue 
replace this line `JSON.parse(localStorage.getItem('bookData')),` with this `JSON.parse(localStorage.getItem('bookData')) || [],`, here you are saying that if there's no data on local storage send empty array.

Reply:
@keroloussamy Thanks for pointing out the issue and that walkthrough will definitely help a lot. 🙏🏾✨